doc/ipython-notebooks/multiclass/KNN.ipynb

```diff
@@ -369,6 +466,10 @@
     "fig = P.figure(figsize=(8,5))\n",
     "P.plot(all_ks, wo_ct['eout'].mean(axis=0), 'r-*')\n",
     "P.plot(all_ks, wo_ct['ein'].mean(axis=0), 'r--*')\n",
+    "P.plot(all_ks, wi_ct['eout'].mean(axis=0), 'b-*')\n",
```

Comment:
I had stretch the figure so the lines wouldn't overlap with each other. See the latest gist in  https://gist.github.com/MikeLing/c57eef21a17c4371e2dcb3f4ce7505eb :)